worker/cb_worker.ml

```diff
@@ -129,7 +129,10 @@ let docker_run ~switch ~log ~docker_config ~cpu img_hash =
     @ run_command ~arch:docker_config.arch
   in
   Lwt_switch.add_hook_or_exec (Some switch) (fun () ->
-      Lwt_unix.system ("docker kill " ^ name) >>= fun _ -> Lwt.return_unit)
+      Lwt_unix.system ("docker ps -q | grep -q " ^ name) >>= function
```

Comment:
Perhaps [`docker ps --filter ...`](https://docs.docker.com/engine/reference/commandline/ps/#filter) can be used instead of grep? :) (and then perhaps with `Lwt_process` to avoid any quoting issues related to executing the command in a shell?)

Reply:
Thanks for the review. I wanted to have a dumb but "visually correct" solution out before I forgot about it. I'll look into doing it correctly later today

Reply:
I am pretty sure `Lwt_unix.system "foo bar"` is equivalent to `Lwt_process.exec (Lwt_process.shell "foo bar")`.
If we don't use `Lwt_process.shell` (to avoid the shell itself), we need to construct the value manually: `Lwt_process.exec ("foo", [|"foo"; "bar"|])` which is quite unwieldy, but maybe it is more correct?

Reply:
@art-w The newest version should be cleaner

Reply:
Thanks!